fe2-android/app/src/main/java/com/github/dedis/student20_pop/model/event/EventType.java

```diff
@@ -3,7 +3,7 @@
 /** Enum class modeling the Event Types */
 public enum EventType {
   ROLL_CALL("R"),
-  ELECTION("E"),
+  ELECTION("Election"),
```

Comment:
This follows the protocol documentation and it is probably not a high priority to address this so close to the end of the semester. Nevertheless it would maybe be good in the future to have some consistency and  change the protocol so that every suffix is spelled out